org-formation/600-access/_tasks.yaml

```diff
@@ -74,6 +74,18 @@ JCWorkflowsNextflowDevAdmin:
     Account: !Ref WorkflowsNextflowDevAccount
     Region: us-east-1
 
+WorkflowsNextflowProdServiceAccount:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/master/templates/IAM/service-account.yaml
+  StackName: workflows-nextflow-prod-service-account
+  Parameters:
+    ManagedPolicyArns:
+      - arn:aws:iam::aws:policy/AdministratorAccess
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: false
+    Account: !Ref WorkflowsNextflowProdAccount
```

Comment:
you can pass a list of accounts and this will create this service account in your prod and dev accounts.

```
Account: [ !Ref WorkflowsNextflowDevAccount, !Ref WorkflowsNextflowProdAccount ]
```

Reply:
Thanks for the tip!